### ISSUE TITLE
Lale-bot to insert comments on user status changes

### ISSUE THREAD

**Issue by dottorer** [lale-help/lale-help#309]
Similar to #298, we also need to track the changes made to a user by Lale-bot comments. This will be shown to admins only as comment in the helper profile page as outlined in #240. 

We will use this for the following changes with the following text:

User activated: "xxxxx actived on DATETIME by yyyy." (xx = user requesting signup, yy = admin that did the approval).
User blocked: "xxxxx blocked on DATETIME by yyyy."
User unblocked: "xxxxx unblocked on DATETIME by yyyy."

**Comment by dottorer** [comment]
@kristysnet Thus one would require the comments function to be added to rhe user profile page visible gor admins only. I will add a new issue for that and link them.

**Comment by dottorer** [comment]
This is done by @kristysnet : we just need translations. Will look into that.

**Comment by dottorer** [comment]
Translation in FR and DE done. Completing.
